### ISSUE TITLE
sample in tutorial gives 0 PSMs

### ISSUE THREAD

**Issue by wsnoble** [crux-toolkit/crux-toolkit#454]
A user pointed out on crux-users that if you run the example from the tutorial here:

http://crux.ms/tutorials/search.html

then you get zero PSMs from Percolator. I suspect that this is because of the recent changes to Tide's default parameters.  (OTOH, in the release notes I see no mention of those changes, so maybe they never made it into the trunk?) We should figure out the source of the problem and perhaps just use a more up-to-date data file.

**Comment by bmx8177** [comment]
As you suspected, this appears to have occurred because of the changes we have changed the default parameters for precursor mass window and mz bin width (April 11th change). 

Couple of options
1) change the documentation
2) switch in a new MS2 file

**Comment by thabangh** [comment]
I think we should just switch to a new ms2 file.  Can you choose one?  It
should be small.

**Comment by bmx8177** [comment]
This new MS2 file will be solely for the tutorial correct? We currently use the same MS2 file for the performance and smoke tests as well.

**Comment by thabangh** [comment]
Yes, just for the tutorial.

**Comment by bmx8177** [comment]
I spent a little time over the holiday to work on this. I have a new MS2 file that will give more than 0 PSMs after the Percolator step. Is it important that we show in this tutorial that Percolator improves things? Currently the example in the documentation (http://crux.ms/tutorials/search.html) show the number of test PSMs increase from 39 to 54). The example I have goes from 110 to 111. It is important to show a larger increase? I am hoping not because creating that MS2 would be more difficult to do.

**Comment by wsnoble** [comment]
No, I don't think that is critical. It would be nice, of course, and if it were easy I would recommend finding an example that shows Percolator off better.  But for now, I think we should just proceed as is.

**Comment by wsnoble** [comment]
We decided that it's OK to use a larger dataset.  User will have to wait a few minutes for results.

**Comment by bmx8177** [comment]
Note for future reference. Using a larger dataset but keeping the same fasta file (small-yeast.fasta). Will replace the contents of demo.ms2 with the new file.

Question for Bill. Do we need the following folder (crux-toolkit/doc/user/demos)? It seems kind of pointless.

**Comment by wsnoble** [comment]
No, we do not need that directory.  Feel free to delete it.

**Comment by bmx8177** [comment]
Bill - I have a patch for this issue. Please let me know who to send it to for code review (github is not letting me upload the txt file for some reason).
